### PR TITLE
docs: allow to use environment variables for limit-count plugin settings

### LINKED ISSUE #10798: docs: add Chinese translation of using variables for secrets in `limit-count` plugin
### Current State

The information in this PR https://github.com/apache/apisix/pull/10607/files is not added to the Chinese docs.

### Desired State

The Chinese docs should have this info as well.

### PR DESCRIPTION
**Pull request by smileby**
### Description

Fixes #10798 

add Chinese translation of using variables for secrets in limit-count plugin

### Checklist

- [X] I have explained the need for this PR and the problem it solves
- [X] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [X] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by lakshya8066** [comment]
@kayx23 Can you help in reviewing this PR?

**Review by kayx23 — CHANGES_REQUESTED** [review]
@smileby please fix for the lint error

**Review by kayx23 — APPROVED** [review]

**Comment by smileby** [comment]
Thank you.  done with the repair.

File: docs/zh/latest/plugins/limit-count.md [hunk 1]
```diff
@@ -250,6 +250,37 @@ curl -i http://127.0.0.1:9180/apisix/admin/routes/1 \
 }'
 ```
 
+此外，插件中的属性值可以引用 APISIX 中的密钥。APISIX 当前支持两种存储密钥的方式 - [环境变量和 HashiCorp Vault](../terminology/secret.md)。
+如果您设置了环境变量 `REDIS_HOST` 和 `REDIS_PASSWORD` ，如下所示，您可以在插件配置中使用它们：
```
Comment (yzeng25) [inline docs/zh/latest/plugins/limit-count.md:254]: @kayx23 @smileby could you help confirm that both the var names should be in lowercase, i.e. `redis_host` and `redis_password`? I can see the original PR https://github.com/apache/apisix/pull/10607/files# mention uppercacse in var values only.

Comment (kayx23) [reply docs/zh/latest/plugins/limit-count.md:254]: Env vars in upper case are correct and expected

Comment (yzeng25) [reply docs/zh/latest/plugins/limit-count.md:254]: ok cool

**Review by yzeng25 — APPROVED** [review]

**Pull request merged**